### PR TITLE
mk_event_epoll: fix uninitialised value

### PR DESCRIPTION
**Pull request by andreswebs**
Fix error:

"Uninitialised value was created by a stack allocation" at `_mk_event_add`

Addresses: #399

### CODE REVIEW

**Review by braydonk — APPROVED** [review]
Thank you for doing this! When I tried this myself I didn't find any adverse effects, but it wasn't 100% thorough; a maintainer should take a look to make sure but it looks safe to me.

**Review by leonardo-albertovich — APPROVED** [review]

**Pull request merged**
